lib/middleware.ts

```diff
@@ -81,11 +98,21 @@ var defaultHandlers = [
   // Response size
   (req, res) => ({"response-size": getResponseSize(res)}),
   // Response time (ns)
-  (req, res) => ({"response-time": getResponseTimeNs(req, res)}),
+  (req, res) => ({value: getResponseTimeNs(req, res)}),
   // Status code
   (req, res) => ({"status-code": res.statusCode}),
   // Ip address
   (req) => ({ip: getIp(req)}),
+
+  // Kayvee's reserved fields
+  // Log level
+  (req, res) => ({level: getLogLevel(req, res)}),
+  // Source
+  // -> Gets passed in among `options` during library initialization
+  // Title
+  () => ({title: "response-time"}),
+  // Type
+  () => ({type: "gauge"}),
```

Comment:
I ended up reverting this change. We don't want this data to get pulled in auto-matically, since we need to be thoughtful about indexing. For now, it's no longer a "gauge". 0584437

When we come up with new ingest approaches, we can update fields as needed.